ome/ome-common-cpp
# Embed release version directly in CMakeLists.txt

See [trello card](https://trello.com/c/vMoR9wUQ/65-release-ome-common-cpp).  Don't get the release version from git.  Continue to use git to set a `-DEV-shortrev` suffix for informational purposes, but don't use git tags when running cmake.

Remove the source archive script.  Update the cmake `source-archive` target to generate a source release directly (this is how the script was invoked by the superbuild).  For actual releasing, we'll run `git archive` directly from the release tag; the script isn't needed for that--it only existed to do the version embedding.

Testing:

- Since this is a leaf dependency, you can build it standalone outside the superbuild for testing
- Check the version is correctly set when running cmake (check build log).
- Check `lib/ome/common/OMECommonConfigVersion.cmake` has the correct package version
- Run `cmake --build . --target source-archive`; it should create a source archive (tar and zip).
- Unpack the archive and try building it.

CI:

- Update OME-COMMON-CPP-release to run `git archive` directly with the release tag.
